quizzes/ch10-02-testing-organization.toml

```
@@ -0,0 +1,55 @@
+[[questions]]
+type = "ShortAnswer"
+prompt.prompt = "When implementing Unit tests what attribute tells Cairo to compile and run the test code only when running `scarb cairo-test`"
+answer.answer = "#[cfg(test)]"
+
+[[questions]]
+type = "MultipleChoice"
+prompt.prompt = "When running integration tests with the file path src/tests/arithmetic_test.cairo, which file contains the module `mod arithmetic_test`?"
+answer.answer = "src/tests.cairo"
+prompt.distractors = [
+	"src/tests/tests.cairo",
+	"src/tests/arithmetic_test.cairo",
+	"src/lib.cairo"
+]
+context = "The `mod arithmetic_test` should be located in the src/tests.cairo file."
+
+[[questions]]
+type = "MultipleChoice"
+prompt.prompt = "How do you run integration tests in a Cairo project"
+answer.answer = "By running `scarb test` with a filter for the integration tests"
+prompt.distractors = [
+	"By running `scarb cairo-test`",
+	"By running `scarb cairo-build` with a filter for the test",
+	"By running `scarb cairo-run`"
+]
+context = """
+To run all integration tests, add to `scarb test` a filter that contains the path with the integration tests"
+"""
+
+[[questions]]
+type = "MultipleChoice"
+prompt.prompt = "What is the difference between unit tests and integration tests in Cairo?"
+answer.answer = "Unit tests test one module at a time, while integration tests test many modules together"
```

Not super accurate: you could have
```rust

mod a {
  fn foo(){}

  mod b {
    fn bar(){}
  }
}
  
#[cfg(test)]
mod tests{
fn test_foo(){}
fn test_bar(){}
```

here you see that i'm actually testing two modules 🤔 

```suggestion
answer.answer = "Unit tests focus on testing individual modules or components in isolation, while integration tests verify the interaction between multiple modules or components."
```

---

The answer have been updated as adviced